indico/migrations/versions/20241023_1517_fb255305ea83_move_user_sessions_to_postgres.py

```diff
@@ -0,0 +1,34 @@
+"""move_user_sessions_to_postgres
+
+Revision ID: fb255305ea83
+Revises: 75db3a4a4ed4
+Create Date: 2024-10-23 15:17:02.490888
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'fb255305ea83'
+down_revision = '34f27622213b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'user_sessions',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('sid', sa.String(), nullable=False, unique=True),
+        sa.Column('data', sa.LargeBinary(), nullable=False),
+        sa.Column('ttl', sa.DateTime(), nullable=False),
+        sa.Column('user_id', sa.Integer(), nullable=False),
+        sa.PrimaryKeyConstraint('id'),
+        sa.ForeignKeyConstraint(['user_id'], ['users.users.id']),
+        schema='indico'
```

Comment:
Would `users` be better here as the db schema instead of indico? I added it in indico since the model was in web.flask.models. but I'm thinking maybe moving it into users?

Reply:
I had the same thought... a `sessions` table in the `users` schema would probably be better